// yahoo-finance2/dist/cjs/src/index-common.js
"use strict";
var __importDefault = (this && this.__importDefault) || function (mod) {
    return (mod && mod.__esModule) ? mod : { "default": mod };
};
Object.defineProperty(exports, "__esModule", { value: true });
// libs
const yahooFinanceFetch_js_1 = __importDefault(require("./lib/yahooFinanceFetch.js"));
const moduleExec_js_1 = __importDefault(require("./lib/moduleExec.js"));
const options_js_1 = __importDefault(require("./lib/options.js"));
const errors_js_1 = __importDefault(require("./lib/errors.js"));
// modules
const autoc_js_1 = __importDefault(require("./modules/autoc.js"));
const historical_js_1 = __importDefault(require("./modules/historical.js"));
const insights_js_1 = __importDefault(require("./modules/insights.js"));
const options_js_2 = __importDefault(require("./modules/options.js"));
const quote_js_1 = __importDefault(require("./modules/quote.js"));
const quoteSummary_js_1 = __importDefault(require("./modules/quoteSummary.js"));
const recommendationsBySymbol_js_1 = __importDefault(require("./modules/recommendationsBySymbol.js"));
const search_js_1 = __importDefault(require("./modules/search.js"));
const trendingSymbols_js_1 = __importDefault(require("./modules/trendingSymbols.js"));
// other
const quoteCombine_js_1 = __importDefault(require("./other/quoteCombine.js"));
console.warn("[yahoo-finance2] Please upgrade to yahoo-finance2@latest, i.e. " +
    "2.x.  The only breaking change is a fix in quoteSummary to " +
    "correctly return Date objects instead of numerical timestamps in " +
    "these submodules: defaultKeyStatistics, earningsHistory, insiderHolders. " +
    "For more info see: " +
    "https://github.com/gadicc/node-yahoo-finance2/commit/4cf1f624d55d92f80db4b36b3afe9414f5eb5a3c");
exports.default = {
    // internal
    _env: {},
    _fetch: yahooFinanceFetch_js_1.default,
    _moduleExec: moduleExec_js_1.default,
    _opts: options_js_1.default,
    // errors
    errors: errors_js_1.default,
    // modules,
    autoc: autoc_js_1.default,
    historical: historical_js_1.default,
    insights: insights_js_1.default,
    options: options_js_2.default,
    quote: quote_js_1.default,
    quoteSummary: quoteSummary_js_1.default,
    recommendationsBySymbol: recommendationsBySymbol_js_1.default,
    search: search_js_1.default,
    trendingSymbols: trendingSymbols_js_1.default,
    // other
    quoteCombine: quoteCombine_js_1.default,
};
